lib/salus/scanners/semgrep.rb

```diff
@@ -39,6 +39,19 @@ def run
           # semgrep has the following behavior:
           #   ouputs a json object with a 'results' and 'errors' key
 
+          if !match['config'].nil?
+            override_keys = %w[pattern language message]
+            override_keys.any? do |k|
```

Comment:
I think you mean `override_keys.each` here?

Reply:
yes, thank you

Reply:
@cbtoni I just addressed the comments and updated error message.  Could you re-review? Thanks